src/libraries/Microsoft.Extensions.Configuration.Json/ref/Microsoft.Extensions.Configuration.Json.csproj

```diff
@@ -6,11 +6,12 @@
     <Compile Include="Microsoft.Extensions.Configuration.Json.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Configuration\ref\Microsoft.Extensions.Configuration.csproj" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsVersion)" />
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Configuration.Abstractions\ref\Microsoft.Extensions.Configuration.Abstractions.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Configuration.FileExtensions\ref\Microsoft.Extensions.Configuration.FileExtensions.csproj" />
+    <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Configuration\ref\Microsoft.Extensions.Configuration.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.FileProviders.Abstractions\ref\Microsoft.Extensions.FileProviders.Abstractions.csproj" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.CompilerServices.Unsafe\ref\System.CompilerServices.Unsafe.csproj" />
```

Comment:
I'm surprised to see Unsafe show up in reference assemblies.  I didn't think it had any exchange types.

Reply:
Also it looks like all these paths are wrong since they omitted `Runtime` from `System.Runtime.CompilerServices.Unsafe`

Reply:
> I'm surprised to see Unsafe show up in reference assemblies. I didn't think it had any exchange types.

I'm unsure what you mean by that. It's coming in as a transitive dependency from System.Memory.

Reply:
> unsure what you mean by that

I don't think any reference assembly in repo should ever have a reference to System.Runtime.CompilerServices.Unsafe since it contains a single static type which will never appear in any other assembly's surface area.

Reply:
If I add a PackageReference to System.Memory or System.Threading.Tasks.Extensions, the reference to System.Runtime.CompilerServices.Unsafe is added transitively. This is already happening in master today. How would I express that a PackageReference should not pass its dependency to the compiler?